service/app/util/AuthenticationHelper.java

```diff
@@ -77,7 +77,7 @@ public static String verifyClientAccessToken(String clientId, String clientToken
     String validClientId = JsonKey.UNAUTHORIZED;
     try {
       Response clientResponse =
-          cassandraOperation.getRecordsByProperties(
+          cassandraOperation.getRecordsByPropertiesWithFiltering(
```

Comment:
I think this function itself can be removed. Not actively used - all consumption clients send us a token and we dont have to support this.

Reply:
Will await for a better day/PR to get this removed.